src/cultionet/scripts/cultionet.py

```diff
@@ -16,11 +16,15 @@
 
 import torch
 import geopandas as gpd
+import pandas as pd
 import yaml
 
 
 logger = logging.getLogger(__name__)
 
+geo_id_data = pd.read_csv('~/geo_id_grid_list.csv')
```

Comment:
Hard-coding this here could be problematic. Checking to see how it's used down below....